typings/index.d.ts

```diff
@@ -275,6 +275,8 @@ declare namespace commander {
     /**
      * Set an alias for the command.
      *
+     * You may call more than once to add multiple aliases, Only the first alias is shown in the auto-generated help.
```

Comment:
```suggestion
     * You may call more than once to add multiple aliases, only the first alias is shown in the auto-generated help.
```
just catched a small typo, a capitalized word after a comma.